api/docker-compose.yaml

```diff
@@ -1,16 +1,22 @@
-version: "3"
+version: "3.4"
+x-api-defaults: &api-defaults
+  build:
+    context: ./src/dev/server
+  image: allofustest/workbench-dev-api:local
+  user: ${UID}
+  working_dir: /w/api
+  environment:
+    - GOOGLE_APPLICATION_CREDENTIALS=/w/api/sa-key.json
+  env_file:
+    - db/vars.env
+  volumes:
+    - src-sync:/w:nocopy
+    - gradle-cache:/.gradle
+    - ~/.config:/.config:cached
+    - ~/.gsutil:/.gsutil:cached
 services:
   scripts:
-    build:
-      context: ./src/dev/server
-    image: allofustest/workbench-dev-api:local
-    user: ${UID}
-    working_dir: /w/api
-    volumes:
-      - src-sync:/w:nocopy
-      - gradle-cache:/.gradle
-      - ~/.config:/.config:cached
-      - ~/.gsutil:/.gsutil:cached
+    <<: *api-defaults
   db:
```

Comment:
There's only one database here, and it feels like a pretty big stretch to think this could be referring to BigQuery. Is there a specific use case you are concerned about? Just reading this file?

Reply:
Fair enough. Just seems like a database should have a name other than database. The confusion isn't so much with this file or even with developers, but when discussing application components with product folks or other stakeholders.